auth/mockServer.go

```diff
@@ -96,18 +100,30 @@ func (m *MockServer) JWKsHandler(w http.ResponseWriter, _ *http.Request) {
 
 // SignToken signs the provided OIDCClaims and header fields into a base64 encoded JWT token signed by the MockServer.
 func (m *MockServer) SignToken(claims OIDCClaims, header map[string]interface{}) (string, error) {
-	token := &jwtgo.Token{
-		Header: header,
-		Claims: claims,
-		Method: jwtgo.SigningMethodRS256,
+	var mapClaims map[string]interface{}
+
+	dataBytes, err := json.Marshal(claims)
```

Comment:
why do you need to marshal and unmarshal it?

Reply:
see also here: https://stackoverflow.com/questions/23589564/function-for-converting-a-struct-to-map-in-golang

Reply:
in the most voted answer it is using another go lib [structs](https://github.com/fatih/structs) to perform conversion to map, I would prefer to stick with json package and use marshaling/unmarshaling instead.